<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuClass</class>
 <widget class="QMainWindow" name="MenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>728</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>隶书</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>无涯</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/无涯.png</normaloff>Resources/无涯.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/Resources/begin ing.png);</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnNew">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>初来乍到</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContinue">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>390</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>人在江湖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetting">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>执掌天下</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>归隐山林</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Menu.qrc"/>
 </resources>
 <connections/>
</ui>
